ground/src/main/java/com/google/android/ground/ui/home/locationofinterestdetails/SubmissionListViewModel.java

```diff
@@ -75,9 +75,12 @@ private Single<List<Submission>> getSubmissions(SubmissionListRequest req) {
       // TODO(#354): Show message or special treatment for layer with no task.
       return Single.just(List.of());
     }
-    return submissionRepository
-        .getSubmissions(req.surveyId, req.locationOfInterestId, req.taskId.get())
-        .onErrorResumeNext(this::onGetSubmissionsError);
+
+    //    return submissionRepository
+    //        .getSubmissions(req.surveyId, req.locationOfInterestId, req.taskId.get())
+    //        .onErrorResumeNext(this::onGetSubmissionsError);
+    // TODO(#1691): Replace the above call with coroutine once migrated to kotlin
+    throw new RuntimeException("Please convert to kotlin and use appropriate method");
```

Comment:
In Kotlin you can also do:
```kotlin
TODO("Convert to Kotlin and use appropriate method")
```

Reply:
This is java codebase. So we can't use this.

Reply:
Sorry I missed that. In that case `UnsupportedOperationException` may be appropriate?

Reply:
Done